tests/reporting/test_market_unit.py

```diff
@@ -0,0 +1,527 @@
+
+def test_market_update_tentative_winning_payout(localFixture, constants, initializeMarket, chain, mockUniverse, mockDisputeBond, mockDisputeBondFactory, mockStakeToken, mockStakeTokenFactory, mockReportingWindow, mockReputationToken, mockAugur):
+    numTicks = 10 ** 10
+    payoutDesignatedNumerators = [numTicks, 0, 0, 0, 0]
+    payoutSecondNumeratorsDispute = [0, numTicks, 0, 0, 0]
+    payoutThridNumerators = [0, 0, numTicks, 0, 0]
+    payoutFourthNumerators = [0, 0, 0, numTicks, 0]
+
+    payoutDesignatedHashValue = initializeMarket.derivePayoutDistributionHash(payoutDesignatedNumerators, False)
+    payoutSecondHashDisputeValue = initializeMarket.derivePayoutDistributionHash(payoutSecondNumeratorsDispute, False)
+    payoutThridHashValue = initializeMarket.derivePayoutDistributionHash(payoutThridNumerators, False)
+    payoutFourthHashValue = initializeMarket.derivePayoutDistributionHash(payoutFourthNumerators, False)
+
+    push_to_designated_report_state(localFixture, mockUniverse, chain, initializeMarket, mockStakeToken, payoutDesignatedNumerators, mockStakeTokenFactory, mockReportingWindow, mockReputationToken, constants)
+    assert initializeMarket.getTentativeWinningPayoutDistributionHash() == initializeMarket.derivePayoutDistributionHash(payoutDesignatedNumerators, False)
+    
+    mockReportingWindow.reset()
+    # get second token
+    mockSecondStakeToken = set_mock_stake_token_value(localFixture, initializeMarket, payoutSecondNumeratorsDispute, mockStakeTokenFactory)
+    
+    # verify stake tokens are set
+    assert initializeMarket.getStakeToken(payoutDesignatedNumerators, False) == mockStakeToken.address
+    assert initializeMarket.getStakeToken(payoutSecondNumeratorsDispute, False) == mockSecondStakeToken.address
+
+    with raises(TransactionFailed, message="tentative winning payout distro can not be 0 we need stake token with positive supply"):
+        mockSecondStakeToken.callUpdateTentativeWinningPayoutDistributionHash(initializeMarket.address, payoutSecondHashDisputeValue)
+    
+    mockStakeToken.setTotalSupply(105)
+    mockSecondStakeToken.setTotalSupply(20)
+    assert initializeMarket.getTotalStake() == 0
+    assert mockSecondStakeToken.callIncreaseTotalStake(initializeMarket.address, 20)
+    assert initializeMarket.getTotalStake() == 20
+    assert mockReportingWindow.getIncreaseTotalStakeCalled() == True
+
+    assert mockSecondStakeToken.callUpdateTentativeWinningPayoutDistributionHash(initializeMarket.address, payoutSecondHashDisputeValue)
+    assert initializeMarket.getTentativeWinningPayoutDistributionHash() == initializeMarket.derivePayoutDistributionHash(payoutDesignatedNumerators, False)
+    assert initializeMarket.getBestGuessSecondPlaceTentativeWinningPayoutDistributionHash() == initializeMarket.derivePayoutDistributionHash(payoutSecondNumeratorsDispute, False)
+    # buy a thrid stake token
```

Comment:
s/thrid/third

Reply:
Texas accent has alway screwed up my spelling, doh!
I'll make the corrections.